### PR TITLE
feat(retrigger-processes): align endurl mapping keys with backend

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

align endurl mapping keys with backend for retriggerable processes

## Why

Apparently the adjustment of the frontend was forgotten in the context of https://github.com/eclipse-tractusx/sig-release/issues/811, with https://github.com/eclipse-tractusx/portal-backend/pull/915 keys were changed that are mapped by the frontend

## Issue

#1266 

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,6 +74,8 @@
   - aligned notification page search field color with other pages [#1178](https://github.com/eclipse-tractusx/portal-frontend/issues/1178)
 - **Home**
   - disabled My Apps button for users without permission [#1165](https://github.com/eclipse-tractusx/portal-frontend/issues/1165)
+- **Application Request**
+  - fixed endurl issue in the retriggering of clearing house step [#1268] (https://github.com/eclipse-tractusx/portal-frontend/pull/1268)
```
Comment (evegufy) [inline CHANGELOG.md:78]: the section wrongly located, in main there's 2.3.0-alpha.3 already released. Please pull the latest state and adjust. Please also put it under Change, according to the PR title and description I just updated.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (manojava-gk) [reply CHANGELOG.md]: updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@
 - **Onboarding Service Provider**
   - removed deprecated page onboardingServiceProvider and every related link [#1261](https://github.com/eclipse-tractusx/portal-frontend/pull/1261)
 
+### Bugfixed
+
+- **Application Request**
+  - fixed endurl issue in the retriggering of clearing house step [#1268] (https://github.com/eclipse-tractusx/portal-frontend/pull/1268)
```
Comment (evegufy) [inline CHANGELOG.md:13]: Please also put it under Change, according to the PR title and description I updated yesterday.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (evegufy) [reply CHANGELOG.md]: only partially addressed

File: CHANGELOG.md (hunk 2 above)
Comment (manojava-gk) [reply CHANGELOG.md]: updated

**Review by evegufy — APPROVED** [review]

**Review by Phil91 — APPROVED** [review]

**Pull request merged**